server/routes/request.js

```diff
@@ -9,7 +9,7 @@ const {
 } = require("../controllers/request");
 
 router.route("/create").post(protect, createRequest);
-router.route("/list").get(protect, userRequests);
+router.route("/list/:currentTime").get(protect, userRequests);
```

Comment:
Why are we passing the current time here as a route parameter here? You can use JS date functions to obtain the current time within the controller instead